package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "searchRelevance",
```

Comment:
@mingkunm can you please update the name field here to "search-relevance-dashboards"